tests/system/gtid-errant/run_condition

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# If this script is present, and it returns zero, the test is executed.
+# Otherwise the test is skipped.
+
+set -e
+
+mysql_version=$(mysql -uci -pci -h 127.0.0.1 --port=10111 -e "select @@global.version" -s -N)
```

Comment:
Is it always guaranteed that server will be using 10111 as the port?

Reply:
yes, this is the starting state for tests (after initial cluster deployment)